docs/concepts/content-addressing.md

```diff
@@ -18,7 +18,22 @@ CIDs are based on the content’s [cryptographic hash](hashing.md). That means:
 
 IPFS uses the `sha-256` hashing algorithm by default, but there is support for many other algorithms. The [Multihash](https://multiformats.io/multihash/) project represents the work for this, with the aim of future-proofing applications' use of hashes and allowing multiple hash functions to coexist. (If you're curious about how hash types in IPFS are decided upon, you may wish to keep an eye on [this forum discussion](https://discuss.ipfs.tech/t/who-decides-what-hashing-algorithms-ipfs-allows/6742).)
 
-## Content Identifiers are not file hashes
+## How CIDs are created
+
+CIDs contain the hash and the codec of the data. In general, the CID is generated for each block by:
+
+1. Computing a cryptographic hash of the block's data.
+1. Combining that hash with codec information about the block using the following <VueCustomTooltip label="A collection of interoperable, extensible protocols for making data self-describable." underlined multiline is-medium>multiformats</VueCustomTooltip>. 
+   - <VueCustomTooltip label="A protocol for differentiating outputs from various well-established hash function." underlined multiline is-medium>Multihash</VueCustomTooltip> for information on the algorithm used to hash the data.
+   - <VueCustomTooltip label="A protocol for differentiating the format of the target data." underlined multiline is-medium>Multicodec</VueCustomTooltip> for information on how to interpret the hashed data after it has been fetched.
+   - <VueCustomTooltip label="A protocol for differentiating the encoding of base-encoded (e.g., base32, base36, base64, base58, etc.) binary data appearing in text." underlined multiline is-medium>Multibase</VueCustomTooltip> for information on how the hashed data is encoded.
```

Comment:
Important to point out that the multibase is only relevant for the string representation of a CID. When in binary format, there's no multibase.

I would also refer to these as standards not protocols